cpp/include/cudf/io/orc_metadata.hpp

```diff
@@ -331,7 +331,7 @@ class orc_metadata {
    * @param num_rows number of rows
    * @param num_stripes number of stripes
    */
-  orc_metadata(orc_schema schema, size_type num_rows, size_type num_stripes)
+  orc_metadata(orc_schema schema, uint64_t num_rows, size_type num_stripes)
```

Comment:
How about `num_stripes`? Should it be 64bit too, according to ORC spec?

Reply:
Specs don't specify the type for the number of stripes. It's just the number of StripeInformation elements in the Footer. I don't think we'll ever encounter a file with more than 2B stripes.